Xamarin.Forms.Core.UnitTests/ShellTests.cs

```diff
@@ -270,7 +265,7 @@ public void CancelNavigation()
 			shell.Items.Add(one);
 			shell.Items.Add(two);
 
-			Assume.That(shell.CurrentState.Location.ToString(), Is.EqualTo("app:///s/one/tabone/content/"));
+			Assume.That(shell.CurrentState.Location.ToString(), Is.EqualTo("//one/tabone/content/"));
```

Comment:
```suggestion
			Assume.That(shell.CurrentState.Location.ToString(), Is.EqualTo("//one/tabone/content"));
```
also below need to fix on the same string